spec/app_spec.rb

```diff
@@ -89,16 +89,14 @@
 
     context 'user options' do
       it 'adds inherited proofing auth_code URL param to authorization endpoint when selected by user' do
```

Comment:
I'm assuming we want to get rid of all references to inherited proofing, so we should remove this test case altogether?